frontend/src/app/shared/services/shopping-list.service.ts

```
@@ -9,6 +9,10 @@ import {
   ListItemRequestDTO,
   ListItemResponseDTO,
 } from '../interfaces/list-item.interface';
+import {
+  ListItemRequestDTO,
```

Biome (2.4.6)

[error] Shouldn't redeclare 'ListItemRequestDTO'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)